tst/EnergyPlus/unit/SimAirServingZones.unit.cc

```diff
@@ -1568,4 +1569,530 @@ TEST_F(EnergyPlusFixture, InitAirLoops_2AirLoop3ADUb)
+        "  10, 10, 2.5,                            !- X,Y,Z Vertex 2 {m}",
+        "  0, 10, 2.5,                             !- X,Y,Z Vertex 3 {m}",
+        "  0, 0, 2.5;                              !- X,Y,Z Vertex 4 {m}",
+
+
+        "ZoneHVAC:EquipmentConnections,",
+        "  Zone1,                                  !- Zone Name",
+        "  Zone1 Equipment List,                   !- Zone Conditioning Equipment List Name",
+        "  Zone1 Inlet Node List,                  !- Zone Air Inlet Node or NodeList Name",
+        "  ,                                       !- Zone Air Exhaust Node or NodeList Name",
+        "  Zone1 Air Node,                         !- Zone Air Node Name",
+        "  Zone1 Return Node List;                 !- Zone Return Air Node or NodeList Name",
+
+        "NodeList,",
+        "  Zone1 Inlet Node List,                  !- Name",
+        "  Zone1 ATU Inlet Node;                   !- Node Name 1",
+
+        "NodeList,",
+        "  Zone1 Return Node List,                 !- Name",
+        "  Zone1 Return Air Node;                  !- Node Name 1",
+
+        "ZoneHVAC:EquipmentList,",
+        "  Zone1 Equipment List,                   !- Name",
+        "  SequentialLoad,                         !- Load Distribution Scheme",
+        "  ZoneHVAC:AirDistributionUnit,           !- Zone Equipment Object Type 1",
+        "  ADU VAV No Rht,                         !- Zone Equipment Name 1",
+        "  1,                                      !- Zone Equipment Cooling Sequence 1",
+        "  1,                                      !- Zone Equipment Heating or No-Load Sequence 1",
+        "  ,                                       !- Zone Equipment Sequential Cooling Fraction Schedule Name 1",
+        "  ;                                       !- Zone Equipment Sequential Heating Fraction Schedule Name 1",
+
+        "ZoneHVAC:AirDistributionUnit,",
+        "  ADU VAV No Rht,                         !- Name",
+        "  Zone1 ATU Inlet Node,                   !- Air Distribution Unit Outlet Node Name",
+        "  AirTerminal:SingleDuct:VAV:NoReheat,    !- Air Terminal Object Type",
+        "  VAV No Rht;                             !- Air Terminal Name",
+
+        // NOTE: Terminal has a min flow of 0
+        "AirTerminal:SingleDuct:VAV:NoReheat,",
+        "  VAV No Rht,                             !- Name",
+        "  Always On Discrete,                     !- Availability Schedule Name",
+        "  Zone1 ATU Inlet Node,                   !- Air Outlet Node Name",
+        "  ATU VAV No Reheat Inlet Node,           !- Air Inlet Node Name",
+        "  1.0,                                    !- Maximum Air Flow Rate {m3/s}",
+        "  Constant,                               !- Zone Minimum Air Flow Input Method",
+        "  0;                                      !- Constant Minimum Air Flow Fraction",               // IMPORTANT
```

Comment:
Important to set the ATU Min Air Flow Fraction to zero, so that it's not the one driving the flow (I got bit by this initially)